packages/rich-text/src/RichTextEditor.tsx

```diff
@@ -140,6 +144,7 @@ const RichTextEditor = (props: Props) => {
             onAction={onAction}
             isDisabled={disabled}
             onChange={setValue}
+            unavailableMarks={unavailableMarks}
```

Comment:
I think `restrictedMarks` may be more descriptive but have no strong opinion. This is a public prop so let's spend a few minutes thinking about a good name

Reply:
👍🏻 to what Ahmed said, I'd suggest `restrictedMarks` or perhaps `prohibitedMarks`

Reply:
`forbiddenMarks`, `marksYouShallNotPass` :mage_man: :laughing: